src/main/drivers/display.h

```diff
@@ -20,6 +20,14 @@
 
 #pragma once
 
+typedef enum {
+    DISPLAYPORT_ATTR_NONE = 0,
+    DISPLAYPORT_ATTR_INFO,
+    DISPLAYPORT_ATTR_WARNING,
+    DISPLAYPORT_ATTR_CRITICAL,
+    DISPLAYPORT_ATTR_BLINK = 0x80, // Device local blink bit or'ed into above
```

Comment:
An enum that combines 'normal' and bit flag values is a bit hard to understand, I'd probably rather define `DISPLAYPORT_ATTR_BLINK` outside of it.

Reply:
Yeah, I was thinking of moving everything to `#define`. Do you think it's better to leave abstract enums and move blink out?